main.go

```diff
@@ -29,6 +29,7 @@ func init() {
 	flag.StringVar(&opts.LogType, "log-type", "simple", "Overwrite to control the type of logs generated. Allowed values: simple, application, synthetic.")
 	flag.StringVar(&opts.LogFormat, "log-format", "default", "Overwrite to control the format of logs generated. Allowed values: default, crio (mimic CRIO output), csv, json")
 	flag.StringVar(&opts.LabelType, "label-type", "none", "Overwrite to control what labels are included in Loki logs. Allowed values: none, client, client-host")
+	flag.BoolVar(&opts.RequireUniqueHostname, "require-unique-hostname", false, "Ensures that the hostname field is unique by adding a random integer to the end.")
```

Comment:
I think this option name is a bit "opaque"... what about calling it "randomize hostname"?

Reply:
On second thought: should it really be "unique"? In that case just adding randomness would help as two random values can still be the same.

Reply:
That's fair. I can change it to reflect this.